app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
Code Efficiency: Please ensure that the 'cp' input command is correct. The current implementation assumes that the command is in the format 'cp source.txt destination.txt'. If the command format is different, the code may not work as expected.